.asf.yaml

```diff
@@ -25,7 +25,8 @@
 #   - The ATC Development mailing list: https://trafficcontrol.apache.org/mailing_lists
 github:
   collaborators:
-  - davidc0le
+  - ocket8888
```

Comment:
@zrhoffman interesting that it adds 2 committers that don't need this status. probably no harm though, right?

Reply:
> probably no harm though, right?

ehh let's find out